src/BaGet.Core/Configuration/BaGetOptions.cs

```diff
@@ -8,6 +8,11 @@ public class BaGetOptions
         /// </summary>
         public string ApiKeyHash { get; set; }
 
+        /// <summary>
+        /// The application root URL for usage in reverse proxy scenarios.
+        /// </summary>
+        public string PathBase { get; set; }

```

Comment:
Let's give the `PathBase` option a default value of `""` so that we can remove the config from `appsettings.json`. I'd like to keep the `appsettings.json` file uncluttered.